### PR TITLE
Requirements for K8s v1.13.12

### PR DESCRIPTION
**Pull request by to-bar**

### CODE REVIEW

File: core/src/epicli/data/common/ansible/playbooks/upgrade.yml [hunk 1]
```diff
@@ -13,19 +13,19 @@
...
+    - { role: upgrade, ver: "1.14.6", cni_ver: "0.7.5" }
 
 
 # latest patch versions:
 # 1.11.10
 # 1.12.10
-# 1.13.8
-# 1.14.4
+# 1.13.12
+# 1.14.8
```
Comment (erzetpe) [inline core/src/epicli/data/common/ansible/playbooks/upgrade.yml]: Why it's different?  1.14.8 and 1.14.6?

File: core/src/epicli/data/common/ansible/playbooks/upgrade.yml [hunk 2]
```diff
@@ -13,19 +13,19 @@
...
-    - { role: upgrade, ver: "1.13.8", cni_ver: "0.7.5" }
+    - { role: upgrade, ver: "1.13.12", cni_ver: "0.7.5" }
 
 - hosts: kubernetes_master:kubernetes_node
   serial: 1
   become: true
   become_method: sudo
   roles:
-    - { role: upgrade, ver: "1.14.4", cni_ver: "0.7.5" }
+    - { role: upgrade, ver: "1.14.6", cni_ver: "0.7.5" }
```
Comment (erzetpe) [inline core/src/epicli/data/common/ansible/playbooks/upgrade.yml]: Why it's different?  1.14.8 and 1.14.6?

File: core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@
   shell: kubectl get nodes -o json
   register: output
   until: output.stdout|from_json|json_query("items[*].status.conditions[?(@.type=='Ready')].status[]")|unique == ["True"]
-  retries: 600
+  retries: 600 # 20min
```
Comment (erzetpe) [inline core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml]: Won't be those comments misleading if someone will forget to change number of minutes after changing retries?

File: core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml [hunk 4]
```diff
@@ -1,9 +1,9 @@
 ---
-- name: Wait for kubectl to find and access a Kubernetes cluster
+- name: Wait for kubectl to find and access K8s cluster
   environment:
     KUBECONFIG: /home/{{ admin_user.name }}/.kube/config
   shell: kubectl cluster-info
-  retries: 50
+  retries: 30 # 1min
```
Comment (erzetpe) [inline core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml]: Won't be those comments misleading if someone will forget to change number of minutes after changing retries?

File: core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml [hunk 5]
```diff
@@ -25,7 +25,7 @@
   shell: kubectl get pods --all-namespaces -o json
   register: output
   until: output.stdout|from_json|json_query('items[*].status.phase')|unique == ["Running"]
-  retries: 150
+  retries: 600 # 20min
```
Comment (erzetpe) [inline core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml]: Won't be those comments misleading if someone will forget to change number of minutes after changing retries?

File: core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml [hunk 6]
```diff
@@ -35,6 +35,6 @@
   shell: kubectl get pods --all-namespaces -o json
   register: output
   until: output.stdout|from_json|json_query('items[*].status.conditions[].status')|unique == ["True"]
-  retries: 600
+  retries: 600 # 20min
```
Comment (erzetpe) [inline core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml]: Won't be those comments misleading if someone will forget to change number of minutes after changing retries?

File: core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/verify.yml [hunk 7]
```diff
@@ -1,46 +1,47 @@
...
+    that: "'{{ version }}' in kubeadm_version.stdout"
+
+- name: verify | Verify node version
   environment:
     KUBECONFIG: /home/{{ admin_user.name }}/.kube/config
   shell: kubectl get nodes {{ inventory_hostname }} -o wide | awk '{print $2" "$5}'
   changed_when: false
   register: get_node_status
+  until: version in get_node_status.stdout
+  retries: 30 # 1min
```
Comment (erzetpe) [inline core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/verify.yml]: Won't be those comments misleading if someone will forget to change number of minutes after changing retries?

File: core/src/epicli/data/common/ansible/playbooks/upgrade.yml (hunk 1 above)
Comment (to-bar) [reply core/src/epicli/data/common/ansible/playbooks/upgrade.yml]: This is just info about the latest, we have 1.14.6 so we should not upgrade to any higher version without updating K8s roles before

File: core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml (hunk 3 above)
Comment (to-bar) [reply core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml]: Comments are for me, someone can remove or update them when making changes in this area. We have code reviews to not forget :)

**Pull request merged**